doc/modules/model_evaluation.rst

```diff
@@ -473,6 +473,12 @@ given binary ``y_true`` and ``y_pred``:
       for C-class classification problem).
     * Macro-average recall as described in [Mosley2013]_ and [Kelleher2015]_: the recall
       for each class is computed independently and the average is taken over all classes.
+    * Class balance accuracy as described in [Mosley2013]_: for each class, the number
+      of correctly predicted samples (diagonal element in the confusion matrix) is normalized
+      by the maximum value of either the total number of observations predicted to the class
```

Comment:
i.e. it's min(precision, recall)?

Reply:
yes, that's how it is defined in the paper.

Reply:
I think you should just state that here... Precision and recall can be assumed knowledge here

Reply:
Or you can reference below.